packages/indexer/native/src/lib.rs

```diff
@@ -586,6 +608,19 @@ declare_types! {
             }
             Ok(emitter.upcast())
         }
+
+        method getBlockEmitter(mut cx) {
+            let mut this = cx.this();
+            let block_emitter = JsBlockEmitter::new::<_, JsBlockEmitter, _>(&mut cx, vec![])?;
+            {
+                let guard = cx.lock();
+                let block_emitter = block_emitter.borrow(&guard);
+                let native_indexer = this.borrow_mut(&guard);
+                let mut block_emitters = native_indexer.block_emitters.write().unwrap();
+                block_emitters.push(block_emitter.clone());
```

Comment:
I think we only need one block emitter, each median time emitter can subscribe from this single block emitter.